Sources/StreamChat/Config/StreamModelsTransformer.swift

```diff
@@ -0,0 +1,77 @@
+//
+// Copyright © 2025 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+/// An object that provides a way to transform Stream Chat models.
+///
+/// Only some data can be changed. The method `replacing()` is used to create a new object from existing data.
+/// All transform functions have default implementation, so you can override only the ones you need.
+///
+/// **Note:** When using `replacing()` method, you need to provide all the properties of the existing object.
```

Comment:
```suggestion
/// - Note: When using `replacing()` method, you need to provide all the properties of the existing object.
```
For nicer formatting in Xcode.